package.json

```diff
@@ -71,8 +73,10 @@
     "jscs": "^2.8.0",
     "jshint": "^2.8.0",
     "mocha": "^2.1.0",
+    "phantomjs": "^1.9.19",
```

Comment:
Right now this is excluded on purpose to be able to test different phantomjs versions. I would prefer to leave it like this.